tests/test_app/App/Model/Table/LettersTable.php

```diff
@@ -9,7 +9,9 @@
  * @link          http://cakephp.org CakePHP(tm) Project
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
-namespace TestApp\Model\Table;;
+namespace TestApp\Model\Table;
+
+;
```

Comment:
Unneeded `;`

Reply:
How about a sniffer for double semicolon?

Reply:
Sounds like an improvement ticket for code-sniffer repo :) But out of scope of this PR then.